modules/space/rendering/renderablestars.cpp

```diff
@@ -266,43 +266,21 @@ namespace {
         openspace::properties::Property::Visibility::User
     };
 
-    constexpr openspace::properties::Property::PropertyInfo RenderMethodOptionInfo = {
-        "RenderMethod",
-        "Render Method",
-        "Render method for the stars",
+    constexpr openspace::properties::Property::PropertyInfo ScaleInfo = {
+        "Scale",
+        "Scale",
+        "A uniform scale factor that determines how much of the total size of the star "
+        "this component is using. If it is 0, it will be hidden, if it is 1, it will "
+        "take the entire size.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
-    const openspace::properties::PropertyOwner::PropertyOwnerInfo
-        UserProvidedTextureOptionInfo =
-    {
-        "UserProvidedTexture",
-        "User Provided Texture",
-        ""
-    };
-
-    const openspace::properties::PropertyOwner::PropertyOwnerInfo
-        ParametersOwnerOptionInfo =
-    {
+    const openspace::properties::PropertyOwner::PropertyOwnerInfo ParametersOwnerInfo = {
         "ParametersOwner",
         "Parameters Options",
```

Comment:
While we're making changes, maybe this one can be given a better name?   Something about size composition?

Reply:
Renamed the container to `SizeComposition` and the dropbox to `Method`